lms/djangoapps/teams/views.py

```diff
@@ -120,7 +120,7 @@ def get(self, request, course_id):
             "teams_detail_url": reverse('teams_detail', args=['team_id']),
             "team_memberships_url": reverse('team_membership_list', request=request),
             "team_membership_detail_url": reverse('team_membership_detail', args=['team_id', user.username]),
-            "languages": settings.ALL_LANGUAGES,
+            "languages": [[lang[0], _(lang[1])] for lang in settings.ALL_LANGUAGES],  # pylint: disable=translation-of-non-string
```

Comment:
@muhammad-ammar How about adding the translated variable in settings just like `ALL_LANGUAGES` ?